docs/deployments/environments.mdx

```diff
@@ -0,0 +1,31 @@
+---
+title: Environments / Instances
+description: An overview of the differences between Development and Production instances
+---
+
+# Environments / Instances
```

Comment:
if we are favouring using "instances" over "environments" in this guide, then let's lead with that for the titles

```suggestion
# Instances / Environments
```